### DISCUSSION TITLE
【建议】：文章列表页头图太大

### DISCUSSION THREAD

**Discussion by Extremeer** [volantis-x/hexo-theme-volantis#745] [General]
### 检查清单

- [X] 已经搜索过，没有发现类似 issue。
- [X] 已经阅读 hexo 官方文档，未发现原生支持。

### 需求描述

【建议】：在文章列表的页面设置头图太大，原文档中说文章添加thumbnail选项会在文章列表中显示，但实际好像没有显示，建议添加一个缩略图在每一个列表的右边。

**Comment by inkss** [comment]
文章列表的页面是指类似首页这种列表页面嘛？

thumbnail 的用法类似于这篇文章 https://volantis.js.org/news/2020-02-18/。

**Reply by Extremeer** [reply]
> 文章列表的页面是指类似首页这种列表页面嘛？

大佬好，是的，thumbnail只能在文章页面显示，想让其在列表页面的每一个列表也显示

**Reply by inkss** [reply]
那得召唤 @xaoxuu 改了，这位大佬审美在线
